src/App.js

```diff
@@ -51,14 +50,19 @@ class App extends Component {
 
   componentDidMount() {
     // example of how to use api requests
-    this.loadPOIs()
     this.loadStories()
     this.loadMaps()
   }
 
   loadPOIs() {
     return Api.getPOIs().then(data =>
-      this.setState({ activeEvents: data, selectedEvent: data[0] })
+      this.setState({ activeEvents: data, selectedEvent: null })
+    )
+  }
+
+  loadPOIsForYear(year) {
```

Comment:
This endpoint also returns the map/year info for the given year right? We will likely use both bits of info (so might need to change the state that is set), and the function name should be changed to reflect that